lib/components/bm-confirm-location-on-map.js

```diff
@@ -0,0 +1,148 @@
+/* @flow */
+'use strict'
+
+const geolocation = require('@blinkmobile/geolocation')
+
+/* :: import type {
+  AngularCallbackOptions, Coordinates, NgModelController
+} from '../../types.js' */
+
+const mod = require('../module.js')
+const utils = require('../utils.js')
+
+class BmConfirmLocationOnMapController {
+  /* :: static $inject : string[] */
+
+  // internal use
+
+  /* :: $digest: () => void */
+  /* :: coords: ?Coordinates */
+  /* :: isEditing: boolean */
+
+  // public attributes (after casts / checks)
+
+  /* :: ngDisabled: boolean */
+  /* :: ngModel: NgModelController */
+  /* :: ngReadonly: boolean */
+
+  constructor ($scope /* : Object */) {
+    this.$digest = $scope.$digest.bind($scope)
```

Comment:
I've switched to `$apply(() => { /* ... */ })` per best-practices rather than using `$digest()` in c2f539b2c1e3e832c640cc4a346435aa212bfd0f